documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_1_Security_Perspective/4_1_2_Identity_and_Trust_Management.md

```diff
@@ -0,0 +1,115 @@
+* **Software Manifests** for the utilized software components which have been evaluated in the Component Certification explained in [Chapter 4.2.4](../4_2_Certification_Perspective/4_2_4_Component_Certification.md)). In addition to the awarded certification levels, the manifests for components with Trust Level 2 and 3 contain verified measurements which can be used to validate that the described software is truly running on the device. To support re-usability of components, the description of each software stack consists of three types of Software Manifests used for describing different layers:
+  * A **Root of Trust for Measurement (RTM) Manifest** for components of the boot stage,
+  * an **Operating System (OS) Manifest** for utilized kernel and user space components, including the container run time enabling the execution of different isolated containers/apps, and
+  * an arbitrary number of **App Manifests** per component identifying the utilized containers/apps.
+Signatures are utilized to represent the passed stages of the certification process (described in [Chapter 4.2.5](../4_2_Certification_Perspective/4_2_5_Processes.md)) and allow a validation of the correctness of the describing artifacts.
+In addition to these static artifacts, the connector operator may add additional attributes to the component's description or have them validated and registered at the DAPS.
+
+All this metadata is provided in machine-readable form. Manifest information for each artifact, such as manifests for all software components, operating system or apps are attached to the artifact itself and will be provided by the IDS Connector that hosts the artifact. Verified Company Descriptions are also provided by IDS Connectors itself, since the ParIS only provides unverified information. Dynamic attributes for each IDS Connector as well as revocation information for Software Manifests are provided by the DAPS.
+
+### Interactions between IDS Connectors and Identity Components
+
+To establish a trusted connection, each connector needs the identity information of the corresponding connector to perform access and usage control decisions. The interactions can be depicted as follows:
+
+![Interaction between IDS Connectors and Identity Components](./media/IdM_Interactions.png)
+#### _Figure 4.1.2.3: Interaction between IDS Connectors and Identity Components_
+
+1. Each IDS Connector acquires a valid identity from the IDS Device CA.
+2. Each IDS Connector requests a current Dynamic Attibute Token from DAPS.
+3. When establishing communication, the DAT of both IDS Connector instances is exchanged. This is also matched with the used TLS certificate.
+
+To avoid the possibility of abusing a DAT by an attacker, these DATs must be treated as confidential information. To further protect from attacks performed with leaked DATs, each Connector has to validate the presented certificate by matching it to the connectors identifier.
```

Comment:
```DATs must be treated as confidential information``` This is simply not possible, because you are sharing your DAT with communication partners.

Reply:
They are confidential. You can decide whom to share your data with :)